emulator/script_test.go

```diff
@@ -110,10 +111,10 @@ func TestExecuteScript_WithArguments(t *testing.T) {
 		require.NoError(t, err)
 
 		scriptWithArgs := `
-			pub fun main(n: Int): Int {
```

Comment:
Maybe revert all the mixed whitespace changes in the PR

Reply:
fixed.